target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -78,6 +78,12 @@ netgear,r6800)
 	ucidef_set_led_netdev "lan3" "LAN3" "white:lan3" "lan3"
 	ucidef_set_led_netdev "lan4" "LAN4" "white:lan4" "lan4"
 	;;
+tplink,archer-a6-v3)
+	ucidef_set_led_netdev "wifi2g" "Wifi 2.4G" "green:wifi2G" "wlan0"
+	ucidef_set_led_netdev "wifi5g" "Wifi 5G" "green:wifi5G" "wlan1"
+	ucidef_set_led_netdev "led_lan" "LAN act" "green:lan" "br-lan" "tx rx"
+	ucidef_set_led_netdev "led_wan" "WAN link" "green:wan" "wan" "link tx rx"
```

Comment:
Well, this is not link, but link+activity.
Is there a specific reason why you use activity only for one and want to use link for the other?

Reply:
Tried to use link for br-lan didnt work. So kept it only for WAN

Reply:
Please don't mark everything as resolved just after you answered, I really hate it if I have to open everything again when looking at a PR.
What do you mean by "didnt work". Is it always off or always on?

Reply:
sorry about that. Never switches ON to indicate link status. For example even a LAN cable is connected to lanX, br-lan shows only activity status.

Reply:
Okay, that's what I would have expected. But then, we can just take the default and use
```
	ucidef_set_led_netdev "lan" "LAN" "green:lan" "br-lan"
	ucidef_set_led_netdev "wan" "WAN" "green:wan" "wan"
```

Reply:
Little confused. Does it mean  the default statements in switch case statement sufficient? There are not default ones present 01_led.sh, also wan LED node is now named as wan_green.. 
OR 
remove last part of the command like "tx rx" and keep only 4 arguments?

Reply:
```
tplink,archer-a6-v3)
	ucidef_set_led_netdev "lan" "LAN" "green:lan" "br-lan"
	ucidef_set_led_netdev "wan" "WAN" "green:wan" "wan"
	;;
```

That's what I want, and that's what all the others do.

The default for the fifth argument is "link rx tx".

Reply:
done!. Thanks for the feedback.